Assets/MixedRealityToolkit.Extensions/SceneTransitionService/SceneTransitionServiceProfile.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions
+{
+    [MixedRealityServiceProfile(typeof(ISceneTransitionService))]
+    [CreateAssetMenu(fileName = "SceneTransitionServiceProfile", menuName = "MixedRealityToolkit/SceneTransitionService Configuration Profile")]
+    public class SceneTransitionServiceProfile : BaseMixedRealityProfile
+    {
+        public bool UseDefaultProgressIndicator => useDefaultProgressIndicator;
+        public GameObject DefaultProgressIndicatorPrefab => defaultProgressIndicatorPrefab;
+        public SystemType CameraFaderType => cameraFaderType;
+
+        public bool UseFadeColor => useFadeColor;
+        public Color FadeColor => fadeColor;
+        public float FadeOutTime => fadeOutTime;
+        public float FadeInTime => fadeInTime;
+        public CameraFaderTargets FadeTargets => fadeTargets;
+
+        [Header("Progress Indicator Options")]
+        [SerializeField]
+        private bool useDefaultProgressIndicator = true;
+
+        [SerializeField]
+        [Tooltip("The prefab used to show progress. Must include a scipt implementing IProgressIndicator.")]
+        private GameObject defaultProgressIndicatorPrefab = null;
+
+        [Header("Fade Options")]
+
+        [SerializeField]
+        private bool useFadeColor = true;
+
+        [SerializeField]
+        private Color fadeColor = Color.black;
+
+        [SerializeField]
+        private float fadeOutTime = 1f;
+
+        [SerializeField]
+        private float fadeInTime = 0.5f;
+
+        [SerializeField]
+        private CameraFaderTargets fadeTargets = CameraFaderTargets.Main;
+
+        [SerializeField]
+        [Implements(typeof(ICameraFader), TypeGrouping.ByNamespaceFlat)]
+        private SystemType cameraFaderType = default(SystemType);
```

Comment:
Nit: Would be great to have tooltips on these and matching doccomments on the public versions of them above.